lib/backend/postgres/index.ts

```diff
@@ -30,8 +36,11 @@ import type {
 import * as utils from './utils';
 export type { StreamChange } from './streams';
 
-// tslint:disable-next-line: no-var-requires
-const { version: coreVersion } = require('../../../package.json');
+// tslint:disable: no-var-requires
+const {
+	name: packageName,
```

Comment:
It took me a bit to understand that these values will be used only on `runDbMigrations` below. 

Maybe using a more specific name would make it more evident? Like `autumndbMigration` instead of `packageName`? 

Or just adding a comment why we're loading these values here ( I assume we want to maintain the `requires` at the top )

Reply:
Added a comment to make it more clear, and yeah I prefer keeping all `require`s and `import`s at the top.